meta/material.go

```diff
@@ -0,0 +1,299 @@
+package meta
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	"strings"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:akso@tcp(127.0.0.1:13306)/akso")
```

Comment:
The tcp connection and user is hard coded, could we use var?

Reply:
I just changed db info to one json file, called mysqlStoreConfig.json